images/vault/tests/main.tf

```diff
@@ -4,18 +4,11 @@ terraform {
   }
 }
 
-variable "digests" {
-  description = "The image digests to run tests over."
-  type = object({
-    vault     = string
-    vault-k8s = string
-  })
+variable "digest" {
+  description = "The image digest to run tests over."
```

Comment:
This is a breaking change for us downstream.  We discussed on slack keeping this as a single module, let's do that unless we have a really good reason not to.

Reply:
Done!